### ISSUE TITLE
Cannot run `useradd` command in a container created by nerdctl+kata.

### ISSUE THREAD

**Issue by zibinpan** [containerd/nerdctl#2921]
### Description

I cannot run the command `useradd` in my host, as seen below:
![image](https://github.com/containerd/nerdctl/assets/79980563/a5a9d979-b4c1-4dde-8209-b0be3e503e71)

For the purpose of learning, I want to create new user in my server. Hence, I use nerdctl + kata container to create a container in rootfull mode. However, in this container, I still cannot run the `useradd` command, as seen below:
![image](https://github.com/containerd/nerdctl/assets/79980563/72f99780-8eed-4b09-8d06-c0389b03ffcc)

So how to fix it? Thank you very much!

### Steps to reproduce the issue

1. run `nerdctl run -it --runtime io.containerd.kata.v2 ubuntu:20.04 sh` to enter a container. 
2. run `su` to enter the root mode in the container.
3. execute `useradd`, and then it raises the problem.

### Describe the results you received and expected

I want to have the totally full root authority in my container created by nerdctl + kata.

### What version of nerdctl are you using?

nerdctl: 1.7.5
kata: 3.3.0

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

None

**Comment by AkihiroSuda** [comment]
> I cannot run the command `useradd` in my host, as seen below:
> ![image](https://github.com/containerd/nerdctl/assets/79980563/a5a9d979-b4c1-4dde-8209-b0be3e503e71)


What is printing this error? Some third-party security software?

**Comment by zibinpan** [comment]
> > I cannot run the command `useradd` in my host, as seen below:
> > ![image](https://private-user-images.githubusercontent.com/79980563/320286373-a5a9d979-b4c1-4dde-8209-b0be3e503e71.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI1ODM0MTksIm5iZiI6MTcxMjU4MzExOSwicGF0aCI6Ii83OTk4MDU2My8zMjAyODYzNzMtYTVhOWQ5NzktYjRjMS00ZGRlLTgyMDktYjBiZTNlNTAzZTcxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA4VDEzMzE1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUwODI5YmYwYTM1Nzg0YmRhMGM1N2JjNWM2MWI1MTFhMzYzMjcyMmNjNzhlMmE2ODU4OWQxZTdiNmY1YmU0MTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.le-Xhfk3EuEKcOvHsId2iy52b_LA03VuZl8kBTG_Lh0)
> 
> What is printing this error? Some third-party security software?

Yeah, it is printed by a higher level tower management system, which limits my host to run some commands like `useradd`. Therefore, I created a container with kata technique and I thought I could have a full control of my container. However, in the created container, I still cannot execute the commands that are forbidden in my host.
